apps/frontend/src/pages/projects/styles.module.css

```diff
@@ -25,3 +25,7 @@
 	line-height: 150%;
 	color: var(--color-text-secondary);
 }
+
+.button-wrapper {
```

Comment:
not sure we need min-width here, on figma this button just takes the size of it's label + paddings